src/XR/features/WebXRLayers.ts

```diff
@@ -118,6 +119,7 @@ class WebXRCompositionLayerRenderTargetTextureProvider extends WebXRLayerRenderT
  * @hidden
  */
 export class WebXRProjectionLayerWrapper extends WebXRCompositionLayerWrapper {
+    public babylonLayer: Nullable<Layer>;
```

Comment:
I didn't even realize we had a Layer type! 
Was this an intentional change? I don't see it being used elsewhere...

Reply:
oh! this is a preparation for a further change I was planning, good catch. I will remove it from here for now.

Reply:
```suggestion
```